ecosystem/sep-0006.md

```diff
@@ -56,13 +121,164 @@ Name | Type | Description
+
+Name | Type | Description
+-----|------|------------
+`asset_code` | string | The code of the asset of interest. E.g. BTC,ETH,USD,INR,etc
+`asset_code` | string | The issuing account id of the asset of interest.
+`account` | string | The stellar account ID involved in the transactions
+`no_older_than` | UTC ISO 8601 string | (optional) The response should contain transactions starting on or after this date & time
+`limit` | int | (optional) the response should contain at most `limit` transactions
+`paging_id` | string | (optional) the response should contain transactions starting prior to this ID (exclusive)
+
+On success the endpoint should return `200 OK` HTTP status code and a JSON object with the following fields:
+
+Name | Type | Description
+-----|------|------------
+`transactions` | array | List of transactions as requested by the client, sorted in time-descending order
+
+Each object in the `transactions` array should have the following fields:
+
+Name | Type | Description
+-----|------|------------
+`id` | string | Unique, anchor-generated id for the deposit
+`status` | string | Processing status of deposit
+`status_eta` | number | (optional) Estimated number of seconds until a status change is expected
+`amount_in` | string | (optional) Amount of deposit as a string with up to 7 decimals that was received by the anchor
+`amount_out` | string | (optional) Amount of deposit as a string with up to 7 decimals that was sent by the anchor to the external stellar account
+`started_at` | UTC ISO 8601 string | (optional) start date and time of transaction
+`completed_at` | UTC ISO 8601 string | (optional) completion date and time of transaction
+`stellar_transaction_id` | string | (optional) transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal
+`external_transaction_id` | string | (optional) ID of transaction on external network that either started the deposit or completed the withdrawal
+
+`status` should be one of:
+
+* `completed` -- deposit fully completed
+* `pending_external` -- deposit has been submitted to external network, but is not yet confirmed. This is the status when waiting on Bitcoin or other external crypto network to complete a transaction, or when waiting on a bank transfer.
+* `pending_anchor` -- deposit is being processed internally by anchor
+* `pending_stellar` -- deposit operation has been submitted to Stellar network, but is not yet confirmed
+* `pending_trust` -- completion of deposit requires user adding trust for the asset
+* `pending_user` -- deposit operation requires a trustline to be processed. Withdrawal operation requires some additional data from user, for bank transfer, for example.
+* `no_market` -- could not complete deposit because no asset/XLM market was available to create the account
+* `too_small` -- Deposit size was less than min_amount
+* `too_large` -- Deposit size exceeded max_amount
+* `error` -- There was an error executing the deposit
```

Comment:
do we need an extra field for error details?

Reply:
Sorry, I mean to include an extra field `message` in the transaction object which should serve this purpose (while also allowing for explaining of other statuses as well).